fakes/fake_git_hub.go

```diff
@@ -111,14 +111,16 @@ type FakeGitHub struct {
 	}
 }
 
-func (fake *FakeGitHub) ListReleases() ([]*github.RepositoryRelease, error) {
+func (fake *FakeGitHub) ListReleases(tagNameRegex string) ([]*github.RepositoryRelease, error) {
 	fake.listReleasesMutex.Lock()
 	fake.listReleasesArgsForCall = append(fake.listReleasesArgsForCall, struct{}{})
 	fake.listReleasesMutex.Unlock()
 	if fake.ListReleasesStub != nil {
 		return fake.ListReleasesStub()
 	} else {
-		return fake.listReleasesReturns.result1, fake.listReleasesReturns.result2
+		matchedReleases, _ := resource.FilterReleasesByTagName(fake.listReleasesReturns.result1, tagNameRegex)
```

Comment:
All the fakes are generated, so this change probably shouldn't be here. They're generated via `go generate ./...` in the root of the repo.